inference-engine/ie_bridges/python/src/openvino/inference_engine/CMakeLists.txt

```diff
@@ -60,13 +60,17 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     target_compile_options(${TARGET_NAME} PRIVATE "-Wno-error=register")
 endif()
 
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
```

Comment:
don't use this package. It does not support `-DPYTHON_EXECUTABLE`

Reply:
done